ruby_event_store/lib/ruby_event_store/mappers/pipeline.rb

```diff
@@ -0,0 +1,24 @@
+module RubyEventStore
+  module Mappers
+    class Pipeline
+      def initialize(transformations)
```

Comment:
@mpraglowski `initialize(*transformation)`?

Reply:
Now with explicit edge transformations it could be defined as:

```ruby
def initialize(*transformations, to_domain_event: DomainEventMapper.new, to_serialized_record: SerializedRecordMapper.new)
  @transformations = ...
end
```

And it will works fine when no edge mappers will be given like:

```ruby
Pipeline.new(Transformation1.new, Transformation2.new)
```

It will work event when we add edge mappers at the end of arguments:

```ruby
Pipeline.new(Transformation1.new, Transformation2.new, to_domain_event: SomeOtherStrategy.new)
```

by it fails with `SyntaxError ((irb):40: syntax error, unexpected ',', expecting =>)` (at least for my tests) when we have definition like:

```ruby
Pipeline.new(to_domain_event: SomeOtherStrategy.new, Transformation1.new, Transformation2.new)
```

Maybe I'm missing something.

Reply:
Keyword arguments go after positional ones.